content/en/docs/contribute/participate/issue-wrangler.md

```diff
@@ -6,25 +6,31 @@ weight: 20
 
 <!-- overview -->
 
-Alongside the [PR Wrangler](/docs/contribute/participate/pr-wranglers),formal approvers, and reviewers, members of SIG Docs take week long shifts [triaging and categorising issues](/docs/contribute/review/for-approvers.md/#triage-and-categorize-issues) for the repository.
+Alongside the [PR Wrangler](/docs/contribute/participate/pr-wranglers), formal approvers,
+and reviewers, members of SIG Docs take week-long shifts
+[triaging and categorising issues](/docs/contribute/review/for-approvers.md/#triage-and-categorize-issues)
+for the repository.
 
 <!-- body -->
 
 ## Duties
 
 Each day in a week-long shift the Issue Wrangler will be responsible for:
 
-- Triaging and tagging incoming issues daily. See [Triage and categorize issues](https://github.com/kubernetes/website/blob/main/content/en/docs/contribute/review/for-approvers.md/#triage-and-categorize-issues) for guidelines on how SIG Docs uses metadata.
+- Triaging and tagging incoming issues daily. See
+  [Triage and categorize issues](/docs/contribute/review/for-approvers.md/#triage-and-categorize-issues)
+  for guidelines on how SIG Docs uses metadata.
 - Keeping an eye on stale & rotten issues within the kubernetes/website repository.
 - Maintenance of the [Issues board](https://github.com/orgs/kubernetes/projects/72/views/1).
 
-### Requirements
+## Requirements
 
 - Must be an active member of the Kubernetes organization.
-- A minimum of 15 [non-trivial](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits) contributions to Kubernetes (of which a certain amount should be directed towards kubernetes/website).
-- Performing the role in an informal capacity already
+- A minimum of 15 [non-trivial](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits)
+  contributions to Kubernetes (of which a certain amount should be directed towards kubernetes/website).
+- Performing the role in an informal capacity already.
 
-### Helpful [Prow commands](https://prow.k8s.io/command-help) for wranglers
+## Helpful [Prow commands](https://prow.k8s.io/command-help) for wranglers
```

Comment:
I think it would be better to remove the link form the heading and add a sentence like "To learn more about the prow commands refer the the prow commands documentation". When clicked on the `Prow commands`  present on the right navigation bar, it takes takes you to the prow commands docs page instead of pointing towards the `Prow commands` section of the docs.

Reply:
Added two sentence in this section after removing the link:

```
Below are some commonly used commands for Issue Wranglers:

-- Commands list --

To find more Prow commands, refer to the [Command Help](https://prow.k8s.io/command-help) documentation.
```